src/main/resources/liquibase/DDP-4702_AT-form.xml

```diff
@@ -0,0 +1,481 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not adding that file yet to the master-changelog.xml till we know it is the final version and will stay that way